tests/unit/TestSessionControllerTest.php

```diff
@@ -0,0 +1,278 @@
+<?php
+
+namespace SilverStripe\TestSession\Tests\Unit;
+
+use DateTime;
+use LogicException;
+use SilverStripe\Control\Director;
+use SilverStripe\Control\Email\Mailer;
+use SilverStripe\Control\HTTPRequest;
+use SilverStripe\Control\Session;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Core\Manifest\ModuleLoader;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\ORM\FieldType\DBDatetime;
+use SilverStripe\TestSession\TestSessionController;
+use SilverStripe\TestSession\TestSessionEnvironment;
+use SilverStripe\TestSession\TestSessionHTTPMiddleware;
+use stdClass;
+use SilverStripe\ORM\Connect\TempDatabase;
+use SilverStripe\Core\Config\Config;
+
+class TestSessionControllerTest extends SapphireTest
+{
+    /**
+     * @var TestSessionEnvironment|PHPUnit_Framework_MockObject_MockObject
+     */
+    private $testSessionEnvironment;
+
+    protected $usesDatabase = true;
+
+    protected function setUp(): void
```

Comment:
yeah, I'm keen on that too, but we still need to support PHP 5.6 for some while.

```php
Parse error: syntax error, unexpected ':', expecting ';' or '{' in /home/vagrant/bench/ins44/vendor/silverstripe/testsession/tests/unit/TestSessionControllerTest.php on line 31 
```